lib/cadet/assets/assets.ex

```diff
@@ -35,19 +32,17 @@ defmodule Cadet.Assets.Assets do
     end
   end
 
-  def list_assets(folder_name, user) do
-    with :ok <- validate_role(user.role),
-         :ok <- validate_folder_name(folder_name) do
+  def list_assets(folder_name) do
+    with :ok <- validate_folder_name(folder_name) do
```

Comment:
In terms of code style, is this fine or should I use a `case` instead of a `with` statement with only 1 match sequence? (as someone who is new to Elixir :pray: )

Reply:
Using `case` is better in this case, yes